includes/core-functions.php

```diff
@@ -108,3 +108,201 @@ function bediq_print_tax_link( $taxonomy ) {
+        'MNT' => 'Mongolia Tughrik',
+        'MOP' => 'Macau Pataca',
+        'MRO' => 'Mauritania Ouguiya',
+        'MUR' => 'Mauritius Rupee',
+        'MVR' => 'Maldives (Maldive Islands) Rufiyaa',
+        'MWK' => 'Malawi Kwacha',
+        'MXN' => 'Mexico Peso',
+        'MYR' => 'Malaysia Ringgit',
+        'MZN' => 'Mozambique Metical',
+        'NAD' => 'Namibia Dollar',
+        'NGN' => 'Nigeria Naira',
+        'NIO' => 'Nicaragua Cordoba',
+        'NOK' => 'Norway Krone',
+        'NPR' => 'Nepal Rupee',
+        'NZD' => 'New Zealand Dollar',
+        'OMR' => 'Oman Rial',
+        'PAB' => 'Panama Balboa',
+        'PEN' => 'Peru Nuevo Sol',
+        'PGK' => 'Papua New Guinea Kina',
+        'PHP' => 'Philippines Peso',
+        'PKR' => 'Pakistan Rupee',
+        'PLN' => 'Poland Zloty',
+        'PYG' => 'Paraguay Guarani',
+        'QAR' => 'Qatar Riyal',
+        'RON' => 'Romania New Leu',
+        'RSD' => 'Serbia Dinar',
+        'RUB' => 'Russia Ruble',
+        'RWF' => 'Rwanda Franc',
+        'SAR' => 'Saudi Arabia Riyal',
+        'SBD' => 'Solomon Islands Dollar',
+        'SCR' => 'Seychelles Rupee',
+        'SDG' => 'Sudan Pound',
+        'SEK' => 'Sweden Krona',
+        'SGD' => 'Singapore Dollar',
+        'SHP' => 'Saint Helena Pound',
+        'SLL' => 'Sierra Leone Leone',
+        'SOS' => 'Somalia Shilling',
+        'SPL*' => 'Seborga Luigino',
+        'SRD' => 'Suriname Dollar',
+        'STD' => 'São Tomé and Príncipe Dobra',
+        'SVC' => 'El Salvador Colon',
+        'SYP' => 'Syria Pound',
+        'SZL' => 'Swaziland Lilangeni',
+        'THB' => 'Thailand Baht',
+        'TJS' => 'Tajikistan Somoni',
+        'TMT' => 'Turkmenistan Manat',
+        'TND' => 'Tunisia Dinar',
+        'TOP' => 'Tonga Pa\'anga',
+        'TRY' => 'Turkey Lira',
+        'TTD' => 'Trinidad and Tobago Dollar',
+        'TVD' => 'Tuvalu Dollar',
+        'TWD' => 'Taiwan New Dollar',
+        'TZS' => 'Tanzania Shilling',
+        'UAH' => 'Ukraine Hryvna',
+        'UGX' => 'Uganda Shilling',
+        'USD' => 'United States Dollar',
+        'UYU' => 'Uruguay Peso',
+        'UZS' => 'Uzbekistan Som',
+        'VEF' => 'Venezuela Bolivar Fuerte',
+        'VND' => 'Viet Nam Dong',
+        'VUV' => 'Vanuatu Vatu',
+        'WST' => 'Samoa Tala',
+        'XAF' => 'Communauté Financière Africaine (BEAC) CFA Franc BEAC',
+        'XCD' => 'East Caribbean Dollar',
+        'XDR' => 'International Monetary Fund (IMF) Special Drawing Rights',
+        'XOF' => 'Communauté Financière Africaine (BCEAO) Franc',
+        'XPF' => 'Comptoirs Français du Pacifique (CFP) Franc',
+        'YER' => 'Yemen Rial',
+        'ZAR' => 'South Africa Rand',
+        'ZMK' => 'Zambia Kwacha',
+        'ZWD' => 'Zimbabwe Dollar'
+    );
+
+    return $currency;
+}
+
+/**
+ * User dropdown depending on a user role
+ *
+ * @param string $role user capability
+ * @return array
+ */
+function bediq_users_dropdown( $role = false ) {
```

Comment:
What does this function do? You could pass a user role, like `administrator`, `editor` and users will be shown from that user role. Instead, you're fetching all the users and looping to check if the user has that cap.